main/main.go

```diff
@@ -121,6 +121,18 @@ var (
 			Value:  "127.0.0.1:8125",
 			EnvVar: "DATADOG_HOST",
 		}),
+		altsrc.NewStringFlag(cli.StringFlag{
+			Name: config.VAR_NAME_DD_AGENT_HOST,
+			Usage: "datadog statsd agent hostname. This value takes precedence over '" + config.VAR_NAME_DD_HOST +
+				"' if it is provided or found in the env",
+			EnvVar: "DATADOG_AGENT_HOST",
```

Comment:
Another option here is to instead make the flag take in the _name_ of the env variable that contains the dd host information, instead of explicitly only checking the  `DATADOG_AGENT_HOST` env var. Wondering if this options make more sense because this is  OSS

Reply:
I think what you have is reasonable. We can phase out the old env vars to use the standard datadog has.